packages/components/src/components/ScalarFloating/ScalarFloating.vue

```diff
@@ -72,15 +72,19 @@ const { floatingStyles, middlewareData } = useFloating(targetRef, floatingRef, {
     :class="{ contents: !!$slots.default }">
     <slot />
   </div>
-  <div
-    ref="floatingRef"
-    class="relative z-context"
-    :style="floatingStyles"
-    v-bind="$attrs">
-    <slot
-      :data="middlewareData"
-      :height="targetHeight"
-      name="floating"
-      :width="targetWidth" />
-  </div>
+  <Teleport
+    v-if="isOpen"
```

Comment:
Do we need to have a v-if for this? I think headless should handle the rendering of the element